packages/portalnetwork/src/client/client.ts

```diff
@@ -427,4 +427,23 @@ export class PortalNetwork extends (EventEmitter as { new (): PortalNetworkEvent
       this.emit('SendTalkResp', src.nodeId, requestId.toString(16), toHexString(payload))
     this.discv5.sendTalkResp(src, requestId, payload)
   }
+
+  public ethGetBalance = async (
```

Comment:
I think we should move all of these `eth` RPC namespace functions to a dedicated subfolder or set of files instead of spreading them across multiple subprotocols.  Thoughts?

Reply:
I dreaming for quite some time already of getting the RPC (base) functionality or at least the framing out of client and maybe here as well and unify and then eventually have a wonderfully typed RPC interface definition for the in-s and out-s which is at the same time something like an implicit (explicit?) RPC spec.

Just dropping here, maybe you guys have thoughts on this as well. 🙂

(might be also already something or a level of extradiction, to *just* have an interface definition in the `Util` package which can then the client and Ultralight implement)